packages/mdc-toolbar/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@material/toolbar",
+  "description": "The Material Components for the web toolbar component",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web.git"
+  },
+  "dependencies": {
+    "@material/elevation": "^0.1.2",
+    "@material/rtl": "^0.1.1",
+    "@material/theme": "^0.1.1",
+    "@material/typography": "^0.1.1"
+  },
+  "publishConfig": {
+    "access": "public"
+  }
+}
```

Comment:
probably because it's a json file and the linter only lints JS, but yeah we try and enforce newline at EOF across all of our files. It's something you can configure in most editors.